lib/lambda_ethereum_consensus/fork_choice/store.ex

```diff
@@ -36,6 +36,11 @@ defmodule LambdaEthereumConsensus.ForkChoice.Store do
     div(time - genesis_time, ChainSpec.get("SECONDS_PER_SLOT"))
   end
 
+  @spec get_current_status_message() :: {:ok, SszTypes.StatusMessage.t()} | {:error, any}
+  def get_current_status_message do
+    get_current_status_message_from_store()
+  end
```

Comment:
Let's merge this with `get_current_status_message_from_store`

Reply:
This way is correct right should i change anythin?